app/views/person_mailer/welcome_email.html.erb

```diff
@@ -2,6 +2,9 @@
 <p/>
 This message contains important information regarding your CMU Silicon Valley email account.
 <p/>
+It is a copy (perhaps outdated) of the information at
```

Comment:
In my opinion, it seems odd to say that something is outdated without updating it. Perhaps it is better to ask the reader to read that information instead? Or to take that information and stick it here instead?